drivers/vulkan/vulkan_context.h

```diff
@@ -184,19 +185,15 @@ class VulkanContext {
 	int command_buffer_count = 1;
 
 	// Extensions.
+	static bool instance_extensions_initialised;
+	static HashMap<CharString, bool> requested_instance_extensions;
+	HashSet<CharString> enabled_instance_extension_names;
 
+	static bool device_extensions_initialised;
```

Comment:
```suggestion
	static bool device_extensions_initialized;
```

Reply:
A Canadian asking an Australian to fix their correct British spelling... :unacceptable: